CI/scripts/test.sh

```diff
@@ -231,8 +231,8 @@ function unit-test-certlib-utility-programs() {
     pushd utilities > /dev/null 2>&1
 
     # Build utilities
-    make -j2 -f cert_utility.mak
-    make -j2 -f policy_utilities.mak
+    make -j${NumMakeThreads} -f cert_utility.mak
+    make -j${NumMakeThreads} -f policy_utilities.mak
```

Comment:
Minor, slightly unrelated change. Fix straggling usages of `-j2` to specify # of threads as determined on host.